.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish
+on:
+  workflow_run:
+    workflows: ["CI"]
+    types:
+      - completed
+  push:
+    branches:
+      - "main"
```

Comment:
should we only release on an actual tagged release instead of every merge to main?

Reply:
One advantage of current approach is that whenever someone raises a PR, we can ask them to mention a changeset and merge would then release a new version without us having to worry about (managing) releases. But it's up to you, I guess it's possible if we do ([ref](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags))
```
push:
    tags:        
      - '**'
```